apps/frontend/src/app/3_organisms/ConversionsHistoryFrame/ConvertHistory.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC, useMemo, useState } from 'react';
+
+import { Select } from '@sovryn/ui';
+
+import { ConvertHistoryType } from './ConvertHistory.types';
+import { convertHistoryOptions } from './ConvertHistory.utils';
+import { AmmConversionsHistoryFrame } from './components/AmmConversionsHistoryFrame/AmmConversionsHistoryFrame';
+import { MyntConversionsHistoryFrame } from './components/MyntConversionsHistoryFrame/MyntConversionsHistoryFrame';
+import { ZeroConversionsHistoryFrame } from './components/ZeroConversionsHistoryFrame/ZeroConversionsHistoryFrame';
+
+export const ConvertHistory: FC = () => {
+  const [selectedHistoryType, setSelectedHistoryType] = useState(
+    ConvertHistoryType.AMM,
+  );
+
+  const selectComponent = useMemo(
```

Comment:
```suggestion
  const SelectComponent = useMemo(
```
It's a component so it should start with a capital letter.